client/src/components/campaigns/CampaignsTable.jsx

```diff
@@ -16,7 +16,7 @@ import {
 import CampaignViewButton from "./CampaignViewButton";
 
 export default function CampaignsTable(props) {
-  const { campaigns } = useContext(CampaignContext);
+  const { campaigns, setCampaigns } = useContext(CampaignContext);
```

Comment:
Not sure u need setCampaigns here unless you're actually using it

Reply:
fixed - good catch